ynput/OpenPype
# Nuke: Expose write knobs - OP-7592

## Changelog Description
This PR adds `exposed_knobs` to the creator plugins settings at `ayon+settings://nuke/create/CreateWriteRender/exposed_knobs`.

When exposed knobs will be linked from the write node to the outside publish group, for users to adjust.

## Testing notes:
1. Add a knob to `ayon+settings://nuke/create/CreateWriteRender/exposed_knobs`.
2. Create write instance in Nuke.
3. Check knob is exposed.
4. Add a different knob to `ayon+settings://nuke/create/CreateWriteRender/exposed_knobs`.
5. Publish and check validate kicks in to add the linked knob.
